frontend/css/layout.css

```diff
@@ -1,7 +1,63 @@
+/*** box sizing border-box for all elements ***/
```

Comment:
Nice to have this added 👍 I would probably place all of this in a separate file since the purpose is a reset rather than specifying the layout for the website.

Reply:
reset.css is now a separate file